[2022-08-02 10:01:31,732] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.data_downloading manual__2022-08-02T10:01:26.637553+00:00 [queued]>
[2022-08-02 10:01:31,753] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Demo.data_downloading manual__2022-08-02T10:01:26.637553+00:00 [queued]>
[2022-08-02 10:01:31,754] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 10:01:31,755] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-02 10:01:31,767] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 10:01:31,843] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): data_downloading> on 2022-08-02 10:01:26.637553+00:00
[2022-08-02 10:01:31,855] {standard_task_runner.py:52} INFO - Started process 4105 to run task
[2022-08-02 10:01:31,865] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Demo', 'data_downloading', 'manual__2022-08-02T10:01:26.637553+00:00', '--job-id', '204', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmplt7rmvh4', '--error-file', '/tmp/tmpet4t9x6c']
[2022-08-02 10:01:31,868] {standard_task_runner.py:80} INFO - Job 204: Subtask data_downloading
[2022-08-02 10:01:32,059] {task_command.py:371} INFO - Running <TaskInstance: Demo.data_downloading manual__2022-08-02T10:01:26.637553+00:00 [running]> on host 31353b71d6cb
[2022-08-02 10:01:32,317] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Demo
AIRFLOW_CTX_TASK_ID=data_downloading
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T10:01:26.637553+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T10:01:26.637553+00:00
[2022-08-02 10:01:32,320] {python.py:173} INFO - Done. Returned value was: 0
[2022-08-02 10:01:32,419] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Demo, task_id=data_downloading, execution_date=20220802T100126, start_date=20220802T100131, end_date=20220802T100132
[2022-08-02 10:01:32,482] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-02 10:01:32,673] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
